deepchecks/vision/utils/classification_formatters.py

```diff
@@ -34,6 +34,23 @@ class ClassificationLabelFormatter(BaseLabelFormatter):
         tensor of shape (N,), When N is the number of samples. Each element is an integer
         representing the class index. A class index is the position of the class in the second dimension of the vector
         of predictions returned by the model.
```

Comment:
I don't understand the last sentence here ("A class index is the position of the class in the...")